src/index.tsx

```diff
@@ -5,6 +5,11 @@ import '@fortawesome/fontawesome-free/css/all.css';
 import './styles/index.scss';
 
 import { App } from './App';
+import { AppProvider } from './components/AppContext';
```

Comment:
Consider using a more specific name for the context provider, like 'AppContextProvider', to improve readability and maintainability. 📚